Source/CarthageKit/ProducerQueue.swift

```diff
@@ -8,50 +8,73 @@
 
 import ReactiveSwift
 
-/// Serializes the execution of SignalProducers, like flatten(.concat), but
-/// without all needing to be enqueued in the same context.
+/// Serializes or parallelizes (up to a limit) the execution of many
+/// SignalProducers, like flatten(.concat) or flatten(.merge), but without all
+/// needing to be enqueued in the same context.
 ///
 /// This allows you to manually enqueue producers from any code that has access
 /// to the queue object, instead of being required to funnel all producers
 /// through a single producer-of-producers.
 internal final class ProducerQueue {
-	private let queue: DispatchQueue
+	private let operationQueue: OperationQueue
```

Comment:
Would you mind creating a new queue class for this?

When we upgrade ReactiveSwift and get `.concurrent()`, we'll want to revert these changes. It'll be easier to do that if it's a separate class.